Testing/functional/tests/cypress/integration/ui/errors/tooManyRequestsAlerts.js

```diff
@@ -0,0 +1,495 @@
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Medication Request", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.enableModules("MedicationRequest");
+        cy.intercept("GET", "**/MedicationRequest/*", {
+            fixture: "MedicationService/medicationRequest.json",
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.intercept("GET", "**/MedicationRequest/*", {
+            statusCode: 429,
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Mobile - COVID-19 Orders", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/Laboratory/Covid19Orders*", {
+            fixture: "LaboratoryService/covid19Orders.json",
+        });
+        cy.enableModules("Laboratory");
+        cy.viewport("iphone-6");
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.intercept("GET", "**/Laboratory/Covid19Orders*", {
+            statusCode: 429,
+        });
+        cy.enableModules("Laboratory");
+        cy.viewport("iphone-6");
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Mobile - Immunization: Unsuccessful Response", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/Immunization?*", {
+            statusCode: 429,
+        });
+        cy.enableModules("Immunization");
+        cy.viewport("iphone-6");
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Mobile - Laboratory Orders", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/Laboratory/LaboratoryOrders*", {
+            fixture: "LaboratoryService/laboratoryOrders.json",
```

Comment:
Do we need to check if LaboratoryOrders are successful before checking LaboratoryOrders again for 429 on line 323?